projects/ceph.yml

```
@@ -0,0 +1,5 @@
+name: Ceph
+url: https://ceph.io/
+modules:
+  - import: https://github.com/ceph/ceph-mixins
```

```suggestion
  - import: github.com/ceph/ceph-mixins
```

For an regex used by the generator to work, `import` must not include the https:
https://github.com/sh0rez/awesome-libsonnet/blob/6f30e73ac603fbb3a41de6c45ccb6e779512ba17/main.go#L109-L112

Also IMHO this is more consistent with the new absolute import paths.